doc/build/dts/api/api.rst

```diff
@@ -422,6 +422,10 @@ device.
      - A node whose ``reg`` is used by the OpenAMP subsystem to determine the
        base address and size of the shared memory (SHM) usable for
        interprocess-communication (IPC)
+   * - zephyr,ipc_shm_translation
+     - A node whose ``reg`` is used by the addr_translation driver to convert,
+       in OpenAMP subsystem, the base address of the shared memory (SHM) from
+       driver to device address, since for some SoCs these are different
```

Comment:
I don't see any DT overlay that uses this in the openamp_rsc_table. Could point it to me or add it if missing, so that we have a better view of the implementation? 
without more details an regarding this description, it seems to me that you are abusing the reg property usage for the memory translation.

Reply:
It's here: https://github.com/zephyrproject-rtos/zephyr/pull/83049/files#diff-1c0848c129df6d76185a95f4b292bea15a94de7708891c87c894e677f8de9e8b
I've added a comment [here](https://github.com/zephyrproject-rtos/zephyr/pull/83049#issuecomment-2650708320) - I can put these 2 PRs together.